omod/src/main/webapp/pages/clinicianfacing/patient.gsp

```diff
@@ -5,52 +5,63 @@
 %>
 <script type="text/javascript">
     var breadcrumbs = [
-        { icon: "icon-home", link: '/' + OPENMRS_CONTEXT_PATH + '/index.htm' },
-        { label: "${ ui.escapeJs(ui.encodeHtmlContent(ui.format(patient.patient))) }" ,
-            link: '${ ui.urlBind("/" + contextPath + dashboardUrl, [ patientId: patient.patient.id ] ) }'}
+        {icon: "icon-home", link: '/' + OPENMRS_CONTEXT_PATH + '/index.htm'},
```

Comment:
Formatting using the OpenMRS style formatter - that is why the initial commit only had changes that I made to the page and nothing else